backend/internal/application/handler/applicationhandler.go

```diff
@@ -28,6 +28,15 @@ import (
 	"sync"
 )
 
+// @title          Application Management API
+// @version.       1.0
```

Comment:
is the version`.` expected ?

Reply:
aa no.. will fix.

Reply:
Fixed with https://github.com/asgardeo/thunder/compare/da228cdeda1a99a78a261e74a95a1596ff29dd75..10f80a6ebf61d3dfc1cd39a65c58c8dc668c0b44